### PR TITLE
Seabeta 199 epb epc

### PR DESCRIPTION
**Pull request by CodrinCotarlanSoftwire**

### CODE REVIEW

File: SeaPublicWebsite/Services/QuestionFlowService.cs [hunk 1]
```diff
@@ -777,6 +791,7 @@ public enum QuestionFlowPage
         ServiceUnsuitable,
         AskForPostcode,
         ConfirmAddress,
+        NoAddressFound,
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Services/QuestionFlowService.cs:794]: These two are essentially identical for all purposes. Both pages are handled by the same 'get' method. Should I just create one `Address` `QuestionFlowPage`?

File: SeaPublicWebsite/Views/EnergyEfficiency/ConfirmAddress.cshtml [hunk 2]
```diff
@@ -25,49 +25,62 @@
...
+                        }
+                        
+                    }
+                })
             }
             else
             {
-                string error = null;
-                foreach (var (_, value) in ViewData.ModelState)
+                ErrorMessageViewModel errorMessageViewModel = null;
```
Comment (DanCorderSoftwire) [inline SeaPublicWebsite/Views/EnergyEfficiency/ConfirmAddress.cshtml:77]: As discussed we can simplify this section quite a lot

File: SeaPublicWebsite/Services/QuestionFlowService.cs (hunk 1 above)
Comment (DanCorderSoftwire) [reply SeaPublicWebsite/Services/QuestionFlowService.cs:794]: As discussed we'll remove the separate "not found" view so these will actually be the same page

File: SeaPublicWebsite/ExternalServices/Models/OpenEpcDto.cs [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 namespace SeaPublicWebsite.ExternalServices.Models
 {
-    public class EpcDto
```
[the remark this answers is not in the document]
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite/ExternalServices/Models/OpenEpcDto.cs:5]: Deleted the old API and forgot about this one :)

File: SeaPublicWebsite/ExternalServices/EpbEpc/EpbEpcApi.cs [hunk 4]
```diff
@@ -61,6 +137,358 @@ private async Task<string> RequestTokenIfNeeded()
...
+        }
+
+        private static GlazingType? GetGlazingTypeFromEpc(EpbEpcAssessmentDto epc)
+        {
+            if (epc.WindowsDescription is null)
+            {
+                return null;
+            }
+
+            // We consider 'mostly' as the point where the number of single-glazed windows is insignificant
```
[the remark this answers is not in the document]
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite/ExternalServices/EpbEpc/EpbEpcApi.cs:471]: Work in progress :)

[the remark this answers is not in the document]
Comment (CodrinCotarlanSoftwire) [reply SeaPublicWebsite/ExternalServices/EpbEpc/EpbEpcApi.cs]: Agreed with Joanna

**Review by DanCorderSoftwire — APPROVED** [review]

**Pull request merged**